packages/core-react/index.d.ts

```diff
@@ -8,4 +8,16 @@ export type TextFieldProps = {
 
 declare const TextField: React.FC<TextFieldProps>;
 
-export { TextField };
+export type ButtonProps = {
```

Comment:
Should we start a `types` folder for the definitions to keep things more organized?

Reply:
yea